contrib/src/main/java/com/datatorrent/contrib/dimensions/AbstractAppDataDimensionStoreHDHT.java

```diff
@@ -127,7 +127,7 @@
   private Unifier<String> queryResultUnifier;
 
   protected long responseDelayMillis;
-  
+
```

Comment:
`git commit --author=<jenkins@datatorrent.com>`. Please open another JIRA. Format changes are not directly related to the upgrade.